react/features/prejoin/components/Prejoin.native.tsx

```diff
@@ -98,20 +90,41 @@ const Prejoin: ({ navigation }: Props) => JSX.Element = ({ navigation }: Props)
         });
     }, [ navigation ]);
 
+    let contentWrapperStyles;
+    let contentContainerStyles;
+    let largeVideoContainerStyles;
+    let toolboxContainerStyles;
+
+    if (aspectRatio === ASPECT_RATIO_NARROW) {
+        contentWrapperStyles = styles.contentWrapper;
+        contentContainerStyles = styles.contentContainer;
+        largeVideoContainerStyles = styles.largeVideoContainer;
+        toolboxContainerStyles = styles.toolboxContainer;
+    } else {
+        contentWrapperStyles = styles.contentWrapperWide;
+        contentContainerStyles = styles.contentContainerWide;
+        largeVideoContainerStyles = styles.largeVideoContainerWide;
+        toolboxContainerStyles = styles.toolboxContainerWide;
+    }
+
     return (
         <JitsiScreen
-            safeAreaInsets = { [ 'right' ] }
             style = { contentWrapperStyles }>
-            <View style = { largeVideoContainerStyles }>
+            <View
+                style = { largeVideoContainerStyles }>
                 <LargeVideo />
             </View>
             <View style = { contentContainerStyles }>
                 <View style = { styles.formWrapper }>
-                    <TextInput
-                        onChangeText = { onChangeDisplayName }
-                        placeholder = { t('dialog.enterDisplayName') }
-                        style = { styles.field }
-                        value = { displayName } />
+                    {
+                        isDisplayNameMandatory
```

Comment:
We should still have the field, but disable the join button in case it's not set, like the lobby screen.